sumo-api/sumo/metrics/evaluator.py

```diff
@@ -282,7 +282,8 @@ def shape_score(self):
         float - shape similarity score
         """
 
-        n_gt = len(self._ground_truth.elements)
+        # number of ground truth elements with evaluated == True
+        n_gt = sum(v.evaluated == True for v in self._ground_truth.elements.values())
```

Comment:
Here and in a few other places you say `v.evaluated == True`, but you should just say `v.evaluated`

Reply:
done.